### PR TITLE
kolibri-server in the TOC

### LINKED ISSUE #108: kolibri-server is not easily found in TOC
The `kolibri-server` section is not easily found in the table of contents because of the deep nesting:

image

https://kolibri.readthedocs.io/en/latest/install.html#kolibri-server-pre-release

Recommend making the heading one level higher, so it's a direct child of Debian/Ubuntu

### PR DESCRIPTION
**Pull request by radinamatic**
Fixes #108

### CODE REVIEW

File: docs/install/_linux.rst [hunk 1]
```diff
@@ -42,7 +42,7 @@ Upgrade
 When you use the PPA installation method, upgrades to newer versions will be automatic, provided there is internet access available.
 
 kolibri-server (pre-release)
-****************************
+----------------------------
```
Comment (benjaoming) [inline docs/install/_linux.rst]: @jredrejo what would you say if we removed "(pre-release)"? Thinking that the latest feedback has been encouraging. You still need the cache stuff to go in?

Comment (radinamatic) [reply docs/install/_linux.rst]: I'm good with removing this if you agree, will amend asap! :+1:

Comment (benjaoming) [reply docs/install/_linux.rst]: Yeah, I would also defend that decision, simply to get a gradually wider adaptation and keeping the huge benefits of this package in mind.

**Review by benjaoming — APPROVED** [review]
Smart simple impro!

**Pull request merged**